infoschema/tables.go

```diff
@@ -1108,14 +1108,38 @@ func GetTiDBServerInfo(ctx sessionctx.Context) ([]ServerInfo, error) {
 			ServerType:     "tidb",
 			Address:        fmt.Sprintf("%s:%d", node.IP, node.Port),
 			StatusAddr:     fmt.Sprintf("%s:%d", node.IP, node.StatusPort),
-			Version:        node.Version,
+			Version:        FormatVersion(node.Version),
 			GitHash:        node.GitHash,
 			StartTimestamp: node.StartTimestamp,
 		})
 	}
 	return servers, nil
 }
 
+// FormatVersion make TiDBVersion consistent to TiKV and PD.
+// The default TiDBVersion is 5.7.25-TiDB-${TiDBReleaseVersion}.
+func FormatVersion(TiDBVersion string) string {
+	var version, nodeVersion string
+
+	// The user hasn't set the config 'ServerVersion',so the TiDBVersion would like "5.7.25-TiDB-TiDBReleaseVersion".
+	if TiDBVersion == fmt.Sprintf("5.7.25-TiDB-%s", mysql.TiDBReleaseVersion) {
```

Comment:
How about use `len(config.ServerVersion) != 0` to check here?

Reply:
Great! Done.